cmd/edit/machinepool/nodepool.go

```diff
@@ -57,7 +57,7 @@ func editNodePool(cmd *cobra.Command, nodePoolID string, clusterKey string, clus
 		os.Exit(1)
```

Comment:
hey @chenz4027 👋 

I'm wondering if we can do a bit of a tidy up whilst we're here and remove the `os.Exit(1)` calls from this function and instead return the `error`. At the minute we simply cannot test this due to those calls being in the function. Do you have time to do that?

Reply:
@robpblake Of course, will work on it